pkg/tnf/handlers/currentkernelcmdlineargs/currentkernelcmdlineargs.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2021 Red Hat, Inc.
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along
+// with this program; if not, write to the Free Software Foundation, Inc.,
+// 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+package currentkernelcmdlineargs
+
+import (
+	"time"
+
+	"github.com/test-network-function/test-network-function/pkg/tnf"
+	"github.com/test-network-function/test-network-function/pkg/tnf/identifier"
+	"github.com/test-network-function/test-network-function/pkg/tnf/reel"
+)
+
+const (
+	successfulOutputRegex = `.+`
+)
+
+// CurrentKernelCmdlineArgs holds information regarding /proc/cmdline.
+type CurrentKernelCmdlineArgs struct {
+	kernelArguments string // Output variable that stores the kernel arguments of the mc
+	result          int
+	timeout         time.Duration
+	args            []string
+}
+
+// NewCurrentKernelCmdlineArgs creates a CurrentKernelCmdlineArgs tnf.Test.
+func NewCurrentKernelCmdlineArgs(timeout time.Duration, mcName string) *CurrentKernelCmdlineArgs {
+	return &CurrentKernelCmdlineArgs{
+		timeout: timeout,
+		result:  tnf.ERROR,
+		args: []string{
+			"cat /proc/cmdline",
+		},
```

Comment:
Please use an actual `[]string{}` string array.  Even though this currently works, the API expects a String array.  So something like this:
```
args: []string{dependencies.CatBinaryName, "/proc/cmdline"}
```

This will make it easier to eventually change this interface, as it is currently bastardized in many places.

Reply:
Now it's fixed